rai/client.go

```
@@ -1256,22 +1256,17 @@ func (c *Client) Execute(
 	// Slow-path
 
 	id := rsp.Transaction.ID
-	count := 0
 	for {
 		txn, err := c.GetTransaction(id)
-		if err != nil {
-			count++
-		}
-
-		if count > 5 {
-			return nil, err
-		}
-
-		if txn.Transaction.State == "COMPLETED" || txn.Transaction.State == "ABORTED" {
-			break
+		if err == nil {
+			if txn.Transaction.State == "COMPLETED" || txn.Transaction.State == "ABORTED" {
+				break
+			}
+		} else {
+			fmt.Println(err)
```

I think we want to return the error, not print it